### ISSUE TITLE
json_library config is required in v0.13.1

### ISSUE THREAD

**Issue by bgentry** [akira/exq#369]
Despite being labeled as a minor bugfix release (the `.1` at the end of the version specifier), this new release caused a breaking change in my app. It appears that its default for `json_library` config does not work properly, because after upgrading my app I encountered a ton of these errors:

```
could not fetch application environment :json_library for application :exq because configuration :json_library was not set
```

Stacktrace:

```
(elixir) lib/application.ex:479: Application.fetch_env!/2
(exq) lib/exq/serializers/json_serializer.ex:21: Exq.Serializers.JsonSerializer.encode!/1
(exq) lib/exq/redis/job_queue.ex:447: Exq.Redis.JobQueue.to_job_serialized/5
(exq) lib/exq/redis/job_queue.ex:22: Exq.Redis.JobQueue.enqueue/6
```

To get things working again I had to add this to my `config.exs`:

```ex
config :exq,
  name: Exq,
  json_library: Jason,
  # ...
```

**Comment by akira** [comment]
@bgentry Sorry for the breakage - I think we missed the default configuration for this on #362 - will need to get a PR on this to fix.

**Comment by akira** [comment]
I released v0.13.2 to fix this, thanks for the heads up!
